spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -3,13 +3,95 @@
 feature 'idv gpo otp verification step', :js do
   include IdvStepHelper
 
-  it_behaves_like 'gpo otp verification step'
-  it_behaves_like 'gpo otp verification step', :oidc
-  it_behaves_like 'gpo otp verification step', :saml
-
-  context 'with GPO proofing disabled it still lets users with a letter verify' do
-    it_behaves_like 'gpo otp verification step'
-    it_behaves_like 'gpo otp verification step', :oidc
-    it_behaves_like 'gpo otp verification step', :saml
```

Comment:
Notes on the changes here:

- I removed the protocol-specific test repetition, consistent with previous changes in #6404 to improve feature spec performance
   - As part of this, I moved the shared example code into this spec file, to be run once.
- The "GPO proofing disabled" context wasn't doing what it claimed to be doing, since it never set up the actual contextual variation. 
   - I added a new "with gpo feature disabled" to the spec file which actually disables the feature to test that a user can still verify their profile.